plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -247,9 +242,8 @@ export const createPublishGitlabMergeRequestAction = (options: {
         ).then((mergeRequest: { web_url: string }) => {
           return mergeRequest.web_url;
         });
-        /** @deprecated */
```

Comment:
I'm thinking if we don't deprecate projectid on the output should it be deprecated on the input above or should that deprecation be removed as well?

Reply:
I think the `projectID` field itself can be deprecated since we are not using the field itself, we are just allowing the use of the GitLab Project ID through query parameters, but we are not using the input variable `projectID` from the webform, if that makes sense.